index.d.ts

```diff
@@ -357,5 +357,11 @@ declare namespace suitest {
 		psVideo?: true,
 	}
 
-	type ScreenOrientationValues = 'portrait' | 'portraitReversed' | 'landscape' | 'landscapeReversed';
+	type ScreenOrientationValues =
+		| 'portrait'
+		| 'portraitReversed'
+		| 'landscape'
+		| 'landscapeReversed'
+		| 'landscapeRight'
+		| 'landscapeLeft';
```

Comment:
We should consider importing @suitest/types and exposing this type from there, but it's OK for now. Something to consider in future